.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: python
+python:
+  - 3.4
+  - 3.5
+install:
+  - pip install .
+  - pip install -e .[tests]
```

Comment:
그게 예전 버전 pip의 버그였던 걸로 기억합니다. 최신 버전에서는 고쳐진 것 같던데요.

On Tue, Aug 16, 2016, 2:47 PM Kang Hyojun <notifications@github.com> wrote:

> In .travis.yml
> <https://github.com/spoqa/nirum-python/pull/16#discussion_r74878986>:
>
> > @@ -0,0 +1,8 @@
> > +language: python
> > +python:
> > +  - 3.4
> > +  - 3.5
> > +install:
> > +  - pip install .
> > +  - pip install -e .[tests]
>
> wheel 패키지때문에 일케 해놓긴했는데 상관없으려나요?
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/spoqa/nirum-python/pull/16/files/f0782b08864b0d8918ef1a2e3cb096bf2fe54a20#r74878986>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAwj38kDnEzimvcbRVyQVlW8pOTOUb7ks5qgU74gaJpZM4Jj2F6>
> .
>